### ISSUE TITLE
Update psutil to the latest version

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2421]
We use psutils (https://github.com/giampaolo/psutil) predominantly for our dashboard as a way, from python, to query running processes and system utilization. As this is a Python library we have take into account our underlying (currently outdated) Python version. However this utilities current version looks to still be compatible with our current python and so it is proposed that we update sooner rather than later, and pin our new version to aid in our pending Python 2 to 3 move.

Current version 3.3.0, proposed version 5.9.4.

**Comment by phillxnet** [comment]
Excluding the eggs package we carry for psutil we have the following code entry:
```
conf/django-hack.py:    join(base, "eggs/psutil-5.9.4-py2.7-linux-x86_64.egg"),
conf/django-hack.py:    join(base, "eggs/psutil-5.9.4-py2.7-linux-aarch64.egg"),
setup.py:        'psutil == 5.9.4',
src/rockstor/smart_manager/data_collector.py:import psutil  # noqa E402
src/rockstor/smart_manager/data_collector.py:            vals = psutil.cpu_times_percent(percpu=True)
```
After the in-development pr to address this issue.

So it looks like we only use this library for our CPU widget on the dashboard.

**Comment by phillxnet** [comment]
Closing as:
Fixed by #2422
